src/dev/flang/fe/LibraryModule.java

```diff
@@ -2382,7 +2382,7 @@ else if (pos == 0)
             var bb = sourceFileBytes(at);
             var ba = new byte[bb.limit()]; // NYI: Would be better if SourceFile could use bb directly.
             bb.get(0, ba);
-            sf = new SourceFile(Path.of(sourceFileName(at)), ba);
+            sf = new SourceFile(Path.of(name() + ".fum").resolve(Path.of(sourceFileName(at))), ba);
```

Comment:
Do not use `".fum"` directly, but `FuzionConstants.MODULE_FILE_SUFFIX` (also at other locations, maybe fix that in another PR).